wikimedia/wikipedia-ios
# Show user education onboarding view on first new topic or comment reply

**Phabricator:** https://phabricator.wikimedia.org/T319075

### Notes
Displays user education onboarding modal on first tap of the new topic button or tapping reply in a thread. This view only shows once. I avoided the attributed string complexity within onboarding body text, but if it's a design requirement after review, I'll adjust to use an iOS 15+ only approach akin to the account vanishing view.

### Test Steps
1. Open any article or user talk page
2. Tap either new topic or reply to a comment
3. Confirm you see user education view. Pulling to dismiss or tapping dismiss button will set the user defaults key to prevent showing this view ever again.
4. Reset user defaults (or just add `UserDefaults.standard.wmf_userHasOnboardedToContributingToTalkPages = false` to the talk page view controller's `viewDidLoad`) and repeat step 2 with the alternate reply type.
